README.md

```diff
@@ -82,52 +82,20 @@ The following instructions follow these notations:
 
 ## Updating the proxy information (Optional)
 
-Update the proxy information specified using the below mentioned properties:
+Update the proxy information in _JAVA_OPTIONS as
+`_JAVA_OPTIONS=-Dhttp.proxyHost=http_proxy_host -Dhttp.proxyPort=http_proxy_port -Dhttps.proxyHost=https_proxy_host -Dhttps.proxyPort=https_proxy_port`, where
 * `http_proxy_host`: Represents the http proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
 * `http_proxy_port`: Represents the http proxy port. Typically, it is the port number in the proxy URL.
 * `https_proxy_host`: Represents the https proxy hostname. Typically, it is an IP address or domain name in the proxy URL.
 * `https_proxy_port`: Represents the https proxy port. Typically, it is the port number in the proxy URL.
 Specify the combination of the hostname and port information together for either http, https or both.
-If no proxy needs to be specified, leave the fields blank.
+If no proxy needs to be specified, do not specify the flags in _JAVA_OPTIONS.
 
-These properties are present in the following script files:
-* <ops-config-dir>/run-ops
-* <ocs-config-dir>/run-ocs
-* <to0scheduler-config-dir>/run-to0scheduler
+## Configuring Ondie ECDSA properties
 
-## Running the SDO IoT Platform SDK demo manually
-
-Update the proxy information (if any) as per the steps outlined previously.
-
-Open a new terminal window and start the rendezvous service:
-```
-$ cd <pri-rendezvous-dir>
-$ ./rendezvous
-```
-
-Open a new terminal window and start the to0scheduler service:
-```
-$ cd <to0scheduler-config-dir>
-$ ./run-to0scheduler
-```
-
-Open a new terminal window and start the ocs service:
-```
-$ cd <ocs-config-dir>
-$ ./run-ocs
-```
-
-Open a new terminal window and start the ops service:
-```
-$ cd <ops-config-dir>
-$ ./run-ops
-```
-
-Open a new terminal window and start the device service:
-```
-$ cd <pri-device-dir>
-$ ./device
-```
+For successful onboarding of Ondie ECDSA clients, appropriate CRLs must be present in the path as specified by the property `org.sdo.ops.ondie-ecdsa-material-path`. 
+If the CRLs needs to be downloaded into the same path, set the property `org.sdo.ops.ondie-ecdsa-material-urls` with a list of appropriate URLs from where the CRL will be downloaded.
```

Comment:
CRLs => CRL files

needs => need

the CRL will be downloaded => the CRL files will be downloaded

Reply:
Updated.